ui/lib/kmip/addon/components/page/credentials-generate.hbs

```diff
@@ -0,0 +1,16 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{#if this.hasGenerated}}
+  <DetailsCredentials @credentials={{@credentials}} @onRevokeCredentials={{this.revokeCredentials}} />
+{{else}}
+  <EditForm
```

Comment:
was previously rendered directly in the `generate.hbs` route template. Now rendered by the component depending on whether or not credentials have generated